clang/test/3C/canwrite_constraints.h

```diff
@@ -19,3 +19,9 @@ int *foo_var = ((void *)0);
 // dedicated test for it.
 inline void no_op() {}
 // CHECK_HIGHER: inline void no_op() _Checked {}
+
+// In the lower case, this should stay wild
+// In the higher case, this should solve to checked
+typedef int* intptr;
+// CHECK_LOWER: typedef int* intptr;
```

Comment:
```suggestion
```
You don't need this check: in the "lower" case, if 3C tried to make any change to this file, it would generate an "unwritable change" error.  For consistency with the other cases, I'd prefer to leave this check out.